helix-view/src/editor.rs

```diff
@@ -908,8 +896,7 @@ pub struct Editor {
     pub auto_pairs: Option<AutoPairs>,
 
     pub idle_timer: Pin<Box<Sleep>>,
-    pub last_motion: Option<Motion>,
-
+    last_motion: LastMotion,
```

Comment:
I think this was clearer before as `Option<Motion>`. It can just be a `type Motion` or `type LastMotion` rather than the newtype it was before.

Reply:
Option<Motion> it is, thanks!